blueprints/component/index.js

```diff
@@ -231,7 +231,7 @@ module.exports = {
 
   normalizeEntityName(entityName) {
     return normalizeEntityName(
-      entityName.replace(/\.js$/, '') //Prevent generation of ".js.js" files
+      entityName.replace(/\.(js|ts)$/, '') //Prevent generation of ".js.js" and ".ts.ts" files
```

Comment:
Why does js.js happen?

Reply:
When people add an extension when generating a component: `ember g component-class foo.js`. I just noticed when copying one of the tests and adjusting it for .ts.

Other ts blueprints will have the same issue.